### PR TITLE
Addextended fab

### PR DESCRIPTION
**Pull request by jaakkojari**
Created M3 ExtendedFAB component, fixed text are label inconsistency and refactored IOBoxButton to use ExtendedFAB

### CODE REVIEW

File: frontend/src/components/TextArea/index.tsx [hunk 1]
```diff
@@ -48,7 +49,7 @@ export const TextArea: React.FC<TextInputProps> = ({
                     'transition-all'
                 )}
             >
-                {touched && value ? label : placeholder}
+                {value ? label : placeholder}
```
Comment (samikuikka) [inline frontend/src/components/TextArea/index.tsx]: Did you pull the latest version of the main?
There should not be any changes to the TextArea.

**Review by samikuikka — CHANGES_REQUESTED** [review]
Some unnecessary changes for example to the text area

File: frontend/src/components/TextArea/index.tsx (hunk 1 above)
Comment (jaakkojari) [reply frontend/src/components/TextArea/index.tsx]: this is the change that fixes the bug related to the output field label text. Should it be in a different branch?

**Comment by samikuikka** [comment]
Now the cypress tests must be fixed

**Review by samikuikka — APPROVED** [review]
now works like it should

**Comment by jaakkoha-bytecraft** [comment]
The two other components could have some comments on what they are.

**Pull request merged**